edb/pgsql/compiler/output.py

```diff
@@ -557,6 +557,14 @@ def serialize_expr_to_json(
             env=env,
         )
 
+    elif irtyputils.is_range(styperef) and not expr.ser_safe:
+        cast_name = s_casts.get_cast_fullname_from_names(
```

Comment:
What's the point of getting the cast_name then?

Reply:
Oops, no point. Fixed.